Documentation/gettingstarted/kind.rst

```diff
@@ -54,13 +54,29 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v2
-   needs to be enabled by setting the kernel ``systemd.unified_cgroup_hierarchy=1`` parameter.
+   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), kind nodes
+   need to have their own cgroup namespaces, and are different from the cgroup namespace
```

Comment:
I think enforcing this might be quite a limitation for some.

I wonder whether we could bring back https://github.com/cilium/cilium/commit/e9ce8306400bf416087046b8d5b013b23ebdcb3e, and make the common root detection more robust.

Another thing is that https://github.com/torvalds/linux/commit/8520e224f547cd070c7c8f97b1fc6d58cff7ccaa made it possible to run the socket-LB on Kind when both cgroup v1 and v2 are enabled. It should be available since v5.14.

Reply:
Ah, good point. I'll add a reference for the kernel fix. Thanks!

>I wonder whether we could bring
It's an option, but newer distros already enable cgroup v2 (so has k8s), and the hard-coded cgroup path can vary depending on the underlying platform. So I'm not sure it's worthwhile.

Reply:
Do you know whether majority of distros has switched to the cgroup v2 with cgroup ns?

Reply:
Here is a list - https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md.